test/test_convergence.jl

```diff
@@ -0,0 +1,213 @@
+import SurfaceFluxes
+SurfaceFluxes.error_on_non_convergence() = false
+using Random
+const rseed = MersenneTwister(0)
+const SF = SurfaceFluxes
+import SurfaceFluxes.UniversalFunctions as UF
+using Statistics
+using StaticArrays
+using Thermodynamics
+using Thermodynamics.TemperatureProfiles
+using Thermodynamics.TestedProfiles
+Thermodynamics.print_warning() = false
+using UnPack
+
+using RootSolvers
+const RS = RootSolvers
+
+using CLIMAParameters
+using CLIMAParameters: AbstractParameterSet
+using CLIMAParameters.Planet
+const CPP = CLIMAParameters.Planet
+const CP = CLIMAParameters
+const SFP = SF.Parameters
+const APS = CP.AbstractParameterSet
+const TP = Thermodynamics.TemperatureProfiles
+import Thermodynamics.TestedProfiles: input_config, PhaseDryProfiles, PhaseEquilProfiles, shared_profiles
+
+include(joinpath(pkgdir(SurfaceFluxes), "parameters", "create_parameters.jl"))
+const FT = Float32;
+toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+param_set = create_parameters(toml_dict, UF.BusingerType())
+thermo_params = SFP.thermodynamics_params(param_set)
+uft = SFP.universal_func_type(param_set)
+
+"""
+    PhaseEquilProfiles(param_set, ::Type{ArrayType})
+
+Returns a `ProfileSet` used to test moist states in thermodynamic equilibrium.
+"""
+function PhaseEquilProfiles(param_set::APS, ::Type{ArrayType}) where {ArrayType}
```

Comment:
Why did we duplicate this?

Reply:
Same as before, we wanted to change this in the next PR.

Reply:
Fixing in #57